webtau-core/src/main/java/com/twosigma/webtau/expectation/equality/handlers/HandlerMessages.java

```diff
@@ -43,28 +47,11 @@ static String expected(CompareToComparator.AssertionMode assertionMode, Object e
     }
 
     static String expected(String prefix, CompareToComparator.AssertionMode assertionMode, Object expected) {
-        String expectedMsg = prefix;
-
-        switch (assertionMode) {
-            case NOT_EQUAL:
-                expectedMsg += "not ";
-                break;
-            case GREATER_THAN:
-                expectedMsg += "greater than ";
-                break;
-            case GREATER_THAN_OR_EQUAL:
-                expectedMsg += "greater than or equal to ";
-                break;
-            case LESS_THAN:
-                expectedMsg += "less than ";
-                break;
-            case LESS_THAN_OR_EQUAL:
-                expectedMsg += "less than or equal to ";
-                break;
-        }
-
-        expectedMsg += expected;
-        expectedMsg += "\n";
-        return expectedMsg;
+        return Stream.of(prefix, assertionMode.getMessage(), expected)
+                .map(Objects::toString)
+                .map(String::trim)
```

Comment:
why trim?

Reply:
Because various places pass in a prefix which already ends in a space.